### ISSUE TITLE
add option : change player UI to old and new

### ISSUE THREAD

**Issue by atuolol** [TeamNewPipe/NewPipe#4794]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
option change look for "player UI". there old UI (with bar) and new UI (no bar)

#### Is your feature request related to a problem? Please describe it
old bar where there is an option to change video resolution, share button, 1 click full screen auto play. i need that.

#### Additional context
help users choose the UI they like

#### How will you/everyone benefit from this feature?
instead of changing the entire player UI with a new look. there should be 2 options to switch to the old UI or the new UI, because each user has different liking. for examples like "i like the old UI especially the old bar". 
sometime i use share button option before play the video. and sometime i want change the resolution before play the video.

**Comment by opusforlife2** [comment]
The old code is gone and will not be added again. Too much has changed and too much time has passed. You can look at #4534 to follow modifications to make the new UI regain functionality of the old one, and #4686 to install the 0.19.8 version with an updated extractor so it works.

**Comment by atuolol** [comment]
> The old code is gone and will not be added again. Too much has changed and too much time has passed. You can look at #4534 to follow modifications to make the new UI regain functionality of the old one, and #4686 to install the 0.19.8 version with an updated extractor so it works.

i don't understand how to modify. I don't understand programing.

sad to hear that. but i understand the situation. thank you for the reply.

**Comment by opusforlife2** [comment]
You don't have to modify anything. xD

I linked the issue so that you can follow updates on the situation, and maybe give your opinion on what looks good and is easier to use. There is a test APK that you can install side by side with your official version to try it out.

In the second linked issue, a developer has provided a 0.19.8 APK with Youtube videos fixed. You can use that as well.

**Comment by atuolol** [comment]
> You don't have to modify anything. xD
> 
> I linked the issue so that you can follow updates on the situation, and maybe give your opinion on what looks good and is easier to use. There is a test APK that you can install side by side with your official version to try it out.
> 
> In the second linked issue, a developer has provided a 0.19.8 APK with Youtube videos fixed. You can use that as well.

it seemed that I didn't read it carefully and was misunderstood

>For now i have a debug build with NewPipe v0.19.8 and Extractor v0.20.2
app-debug.zip
The changes needed for this one are very minor: XiangRongLin@bc71819

if you mean using app-debug then it doesn't work. I uninstalled the latest 0.20.2 and it's still the same, not installed.



i have an idea, what about the same UI. the buttons like (close button, share, change resolution, etc.) can be appearing over the thumbnail before the video plays. 🤔
with it still helping me to share video to browser and view the reply or change the video resolution or close the video immediatelly. 🤩

in short, add options. player button can appear before video plays + "thumbnail show but hidden button" like now for others who like it.

**Comment by atuolol** [comment]
or the user must tap the screen to bring up the button on the player > then tap the play button and the video runs. 🤔

**Comment by opusforlife2** [comment]
Have a look at the previous comments on the linked issue and post your ideas there, not here.
